### DISCUSSION TITLE
EmailIcon does not exist in the npm package

### DISCUSSION THREAD

**Discussion by Pham-dev** [twilio-labs/paste#2670] [Request: Icon] [answered]
Hi y'all.  I was trying to use the EmailIcon from the Icon list in paste but it does not seem to exist.  In the documentation it does but I am also using the latest version of @twilio-paste/icons which is 9.0.0.  

My package Versions:
"@twilio-paste/core": "15.1.0",
"@twilio-paste/icons": "9.0.0",
"prop-types": "^15.7.2",
"react": "^17.0.2",
"react-dom": "^17.0.2",

**Comment by SiTaggart** [comment]
Hi @Pham-dev 

We're just in the middle of cutting a release to NPM now. You were unlucky enough and quick enough to see the in-between state of merged worked and pre-publshed packages.

I'll update here once it hit NPM.

**Reply by Pham-dev** [reply]
Thanks for the quick reply.  I'll be on the look out for the update :)

**Comment by SiTaggart** [comment, marked as answer]
Hi @Pham-dev 

The Email Icon is now in NPM https://paste.twilio.design/components/icons
